mmrazor/models/task_modules/estimators/counters/flops_params_counter.py

```diff
@@ -0,0 +1,467 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import sys
+from functools import partial
+
+import torch
```

Comment:
Unify `units` for params and flops under the scope of `flops_params_counter.py`

Reply:
[cancel unit convert in accumulate_sub_module_flops_params](https://github.com/open-mmlab/mmrazor/pull/230/commits/335e9c4d17f91e09039b047f7b0d55cc9b22683c), remain the rest as the origin version.